config/logging.yaml

```diff
@@ -62,3 +67,4 @@
 :test:
   :level: debug
   :filename: "test.log"
+  :truncate: true
```

Comment:
This is still only enabled for the test environment, but mostly should have worked from the test_helper before now.